src/main/java/com/org/jenkins/Custom/Jenkins/Distribution/Service/UpdateCenterService.java

```diff
@@ -0,0 +1,27 @@
+package com.org.jenkins.Custom.Jenkins.Distribution.Service;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Package name contains upper case characters](https://app.codacy.com/gh/jenkinsci/custom-distribution-service/pullRequest?prid=5630399)

Reply:
I guess we can just ignore this for now. Because the changes have already been made in the other PR #28